tests/test_dcos_e2e/backends/docker/test_docker.py

```diff
@@ -517,3 +517,36 @@ def test_pass_bridge(self) -> None:
             bridge_ip_address = networks['bridge']['IPAddress']
             assert bridge_ip_address == str(master.public_ip_address)
             assert bridge_ip_address == str(master.private_ip_address)
+
+
+class TestOneMasterHostPortMap:
+    """
+    Tests for setting host port map on a master Docker container.
+    """
+
+    def test_one_master_host_port_map(self) -> None:
+        """
+        It is possible to expose admin router to a host port.
+        """
+
+        with Cluster(
+            cluster_backend=Docker(
+                one_master_host_port_map={'80/tcp': 8000},
+                transport=Transport.DOCKER_EXEC,
+            ),
+            masters=3,
+            agents=0,
+            public_agents=0,
+        ) as cluster:
+            port_map_found = False
+            for node in cluster.masters:
+                container = _get_container_from_node(node=node)
+                ports = container.attrs['HostConfig']['PortBindings']
+                if ports is None:
+                    continue
+                if '80/tcp' in ports:
+                    host_ports = ports['80/tcp']
+                    if host_ports[0]['HostPort'] == '8000':
+                        port_map_found = True
+                        break
```

Comment:
❓ Remove this `break` else you might not hit 100% coverage - the invisible "else" might not be hit if the first node you check is `master-0`.